### DISCUSSION TITLE
How to remove files

### DISCUSSION THREAD

**Discussion by gldtn** [NeogitOrg/neogit#1275] [Q&A] [answered]
How do I remove tracked files? I only see a list of Untracked files. 

Thanks!

**Comment by CKolkey** [comment]
The status buffer shows _changes in git_ not files. An untracked file will have its entire content as a change, since it didn't exist before, whereas a tracked file will only show its changed hunks.

To remove a tracked file, you could use a tool meant for filesystem manipulation

**Reply by gldtn** [reply]
Understood, but I'm not sure if I expressed myself correctly.. in git I can do this `git rm -r --cached lua/plugins/lsp/disabled/` so it stops tracking that file/folder, so if I understood correctly I can't do that directly in Neogit, and I would need to use something like neo-tree?

Thanks!

**Reply by CKolkey** [reply]
Good idea - added here: [`788e501` (#978)](https://github.com/NeogitOrg/neogit/pull/978/commits/788e501070be6100fb565a994bede318700cba00)

It's bound to `K` on the status buffer.

**Reply by gldtn** [reply]
I guess all I'm missing now is a way to show tracked files in the status buffer, for example:

```lua
 sections = {
        tracked = {
          folded = true,
          hidden = false,
        },
}
```
I tried the above which did not throw any errors, so I'm assuming it's there, but just not showing up?

That way I can use `K` on the files/folder, without being able to see the tracked files I'm still a bit lost on how to use the new bound key.

Thanks for your patience!

**Reply by CKolkey** [reply]
Ah, maybe there's a misunderstanding. There is no "tracked" section - The status buffer is essentially a projection of "git status" with extra bits. It's not a file browser. So if it doesn't show up when you run "git status" it doesn't show up there either.

I'll see about adding a thing for `K` in the next few days.

**Reply by CKolkey** [reply, marked as answer]
Alright, I think I made it a lot better. `K` will now launch a fuzzy finder with a list of _all_ tracked files, and any files selected will be on the top of the list. Multiple files can be selected :)

**Reply by gldtn** [reply]
Your awesome, exactly what I was looking for. 

Thanks for integrating this feature!

**Reply by CKolkey** [reply]
Thanks for asking :)

**Reply by gldtn** [reply]
> Multiple files can be selected :)

What is the key to select multiple files? I tried tab which I believe is the default for fzf-lua, and a few others but can't figure it out.

**Reply by CKolkey** [reply]
Oh, I use telescope, so multi-select is only implemented if you use that. The mini.pick and fzf-lua integrations were community provided, so some features are not available. I'm open to a PR to add it, but otherwise thats left to users to deal with.

You can see the differences here: https://github.com/NeogitOrg/neogit/blob/master/lua/neogit/lib/finder.lua

**Reply by gldtn** [reply]
It was a lot easier than I thought.. PR sent.

Thank you!
